src/main/cpp/controllers/turret/ManualAim.cpp

```diff
@@ -0,0 +1,35 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "controllers/turret/ManualAim.h"
+#include "subsys/MechanismFactory.h"
+#include "subsys/IMechanism.h"
+#include "subsys/MechanismParameters.h"
+#include "subsys/Turret.h"
+#include "controllers/ControlData.h"
+ManualAim::ManualAim(ControlData* controlData, double mechData) : m_controlData(controlData),
+    m_mechData(mechData), 
+    m_atTarget(false)
+{
+    auto factory = MechanismFactory::GetMechanismFactory();
+    m_turret = factory->GetIMechanism(MechanismTypes::TURRET);
+}
+
+void ManualAim::Init()
+{
+
+}
+
+void ManualAim::Run()
+{
```

Comment:
should add code to call TeleopControl::GetInstance()->GetAxisValue( TeleopControl::FUNCTION_IDENTIFIER::SHOOTER_MANUAL_AIM ); to get the value to move.